app/views/layouts/_header.html.erb

```diff
@@ -0,0 +1,40 @@
+<nav class="navbar navbar-expand-md navbar-light navbar-laravel">
+  <div class="container">
+    <a class="navbar-brand" href="http://127.0.0.1:8000">
+    Laravel
+    </a>
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
```

Comment:
kiểm tra lại toàn bộ các file erb và bỏ các thuộc tính bắt đầu bởi `aria-`
Optional: Tại sao không cần thuộc tính này nhỉ?

Reply:
thuộc tính này thêm vào để hộ trợ giải thích nội dung web